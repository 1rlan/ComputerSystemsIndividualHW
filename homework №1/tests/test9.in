20 -1325131357
-1033312463 -1325131357 -225715071 -670906435 -634110613 -1325131357 -461881150 -1325131357 -1129821243 -1590663081 -972963408 -185396848 -1325131357 -108217640 -1195207214 -1325131357 -1849258045 -1929512300 -1325131357 -274532261 
