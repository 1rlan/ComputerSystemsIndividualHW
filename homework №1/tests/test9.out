-1033312463 -225715071 -670906435 -634110613 -461881150 -1129821243 -1590663081 -972963408 -185396848 -108217640 -1195207214 -1849258045 -1929512300 -274532261
